python-backend/app/api/incidents/resources/incidents_resource.py

```diff
@@ -128,6 +132,10 @@ def _apply_filters_and_pagination(self, args):
 
         # Apply sorting
         if args['sort_field'] and args['sort_dir']:
+            logging.warning(f'the sort dir is: { args["sort_dir"]}')
+            logging.warning(f'the sort field is: { args["sort_field"]}')
+            logging.warning(f'the sort field is: { sort_field[args["sort_field"]]}')
+            logging.warning(f'the sort model is: { sort_models[args["sort_field"]]}')
```

Comment:
Please remove all the logging in this file (and the logging import) 🙂

Reply:
removed